android/domain/src/main/java/co/nimblehq/smsforwarder/domain/usecase/GetFiltersUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.smsforwarder.domain.usecase
+
+import co.nimblehq.smsforwarder.domain.data.error.DataError.GetDataError
+import co.nimblehq.smsforwarder.domain.persistence.FiltersPersistence
+import co.nimblehq.smsforwarder.domain.schedulers.BaseSchedulerProvider
+import co.nimblehq.smsforwarder.domain.usecase.base.CompletableUseCase
+import io.reactivex.Completable
+import javax.inject.Inject
+
+class GetFiltersUseCase @Inject constructor(
+    schedulerProvider: BaseSchedulerProvider,
+    private val persistence: FiltersPersistence
```

Comment:
I think we should swap the arguments a bit, the private val should be first.

Reply:
@lydiasama This is original order following our android-template 🙏

Reply:
I see.